src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final byte dateIndex = 0;
+    static final byte nameIndex = 1;
+    static final byte workingDaysIndex = 2;
+    static final byte hourlyRateIndex = 3;
+    static final DateTimeFormatter formatter =
```

Comment:
example of constant + what about access modifiers?
![Screenshot 2022-07-19 at 07 35 15](https://user-images.githubusercontent.com/60258373/179665517-b9faaf45-c544-4bc5-bd41-f7c3594fc783.png)